rwviewer/ViewerWidget.cpp

```diff
@@ -163,7 +165,11 @@ void ViewerWidget::drawFrameWidget(ModelFrame* f, const glm::mat4& m) {
         glLineWidth(1.f);
     }
     dp.textures = {whiteTex};
-    renderer->getRenderer()->drawArrays(thisM, _frameWidgetDraw, dp);
+    
+    RW_CHECK(renderer != nullptr, "GameRenderer is null");
+    if(renderer != nullptr) {
+        renderer->getRenderer()->drawArrays(thisM, _frameWidgetDraw, dp);
+    }
```

Comment:
This should probably just error if the GameRenderer is null.

(I think the viewer tools is dead anyway :P )